src/Compilers/VisualBasic/vbc/Program.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using Microsoft.CodeAnalysis.BuildTasks;
+using System;
```

Comment:
It can, but you have to set the "Add System first" option (which is off by default)